lib/SlackBot.js

```diff
@@ -2,6 +2,7 @@ var Botkit = require(__dirname + '/CoreBot.js');
 var request = require('request');
 var express = require('express');
 var bodyParser = require('body-parser');
+var querystring = require('querystring');
```

Comment:
Will do. As a new Node.js developer, can you help me understand why this needs to be in the package.json?